### ISSUE TITLE
Does not detect tsx when imported using --import

### ISSUE THREAD

**Issue by gajus** [webpro/knip#351]
When we updated our `package.json`:

```diff
- "dev:server": "tsx ./server/server.ts",
+ "dev:server": "node --import tsx ./server/server.ts",
```

Knip started to report `tsx` as unused.

**Comment by webpro** [comment]
:rocket: _This issue has been resolved in v2.41.4. See [Release 2.41.4](https://github.com/webpro/knip/releases/tag/2.41.4) for release notes._

**Comment by webpro** [comment]
:rocket: _This issue has been resolved in v3.0.0-canary.4. See [Release 3.0.0-canary.4](https://github.com/webpro/knip/releases/tag/3.0.0-canary.4) for release notes._
